### PR TITLE
Vb 3114 orchestration service changes to use the new endpoint as only name is needed

### PR DESCRIPTION
**Pull request by dhirajshettymoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/PrisonRegisterClient.kt [hunk 1]
```diff
@@ -14,18 +14,9 @@ class PrisonRegisterClient(
...
 ) {
   fun getPrisons(): List<PrisonDto>? {
-    return webClient.get().uri("/prisons")
+    return webClient.get().uri("/prisons/names")
       .retrieve()
       .bodyToMono<List<PrisonDto>>()
       .block(apiTimeout)
   }
-
-  fun getPrison(
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/PrisonRegisterClient.kt:23]: was not being used.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/prison/register/PrisonDto.kt [hunk 2]
```diff
@@ -6,9 +6,6 @@ import io.swagger.v3.oas.annotations.media.Schema
 data class PrisonDto(
```
Comment (AledWEvans) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/prison/register/PrisonDto.kt:6]: I would call it PrisonNamesDto

**Review by AledWEvans — COMMENTED** [review]
one comment please update

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/prison/register/PrisonDto.kt (hunk 2 above)
Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/prison/register/PrisonDto.kt]: Updated Dto name and some accompanying logic

**Pull request merged**